R/PipeOpFFS.R

```diff
@@ -159,11 +184,15 @@ make_fextractor = function(f) {
       upper = interval[[2L]]
 
       if (is.na(lower) || is.na(upper)) {
-        NA_real_ # no observation in the given interval [left, right]
+        rep(NA_real_, length(features)) # no observation in the given interval [left, right]
```

Comment:
this should not be `length(features)` I think

Reply:
Also there is a lot of duplicated code for the irreg and reg case

Reply:
you need length(features), since you need the same number of NAs as features

Reply:
I don't see much room of beneficial abstraction, however I've applied one change already